### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by wildcard329**

### CODE REVIEW

File: src/helpers/RouteHelper.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+//         <Route {...rest} render={() => (
+//           authStatus === null ? <CircularProgress  color="primary"/> :
+//           authStatus === true
+//           ? <Redirect to='/dashboard'/>
+//           : <Component/>
+//         )}/>
+//       )
+// }
+
+// export default RouteHelper;
```
Comment (alexparker) [inline src/helpers/RouteHelper.js:29]: Don't commit this if it's not doing anything.

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
```
Comment (alexparker) [inline src/index.css:13]: What is this file and how is it separate/different from the App.less(App.css) files?

File: src/less/App.less [hunk 3]
```diff
@@ -0,0 +1,7 @@
+@import "./global/normalize.less";
+@import "./global//fonts.less";
+@import "./global/variables.less";
+@import "./global/mixins.less";
+@import '~antd/dist/antd.css';
```
Comment (alexparker) [inline src/less/App.less:5]: does import order matter here? Wondering if antd needs to be before any of the mixins or variables if they depend on  antd? if not, disregard

File: src/view/dashboard/containers/JobListings.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Layout, Menu, Breadcrumb } from 'antd';
+import Title from 'antd/lib/typography/Title';
+import '../../../App.css'
```
Comment (alexparker) [inline src/view/dashboard/containers/JobListings.jsx:4]: May want to look into **babel resolvers** or **webpack aliases** so you can avoid these deep pathed references.
One example: https://webpack.js.org/configuration/resolve/#resolvealias

File: src/view/dashboard/containers/JobTracker.jsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+//                   <JobBoard id='board-6' className='board' />
+//               </div>
+//             </Content>
+//           </Layout>
+//         </Layout>
+//         </div>
+//     )
+// }
+
+// export default JobTracker;
```
Comment (alexparker) [inline src/view/dashboard/containers/JobTracker.jsx:66]: Again, don't commit commented out files, just commit the deletion of the file, or get familiar with git stashing

File: src/view/dashboard/containers/Login.jsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
+// Core
+import React from 'react'
+import { useSelector } from 'react-redux';
+import { useOktaAuth } from '@okta/okta-react'; 
+import { Button } from 'antd';
+
+// Model && Helpers
+import LoginController from '../../../controllers/LoginController';
```
Comment (alexparker) [inline src/view/dashboard/containers/Login.jsx:8]: Above referenced webpack aliases would give you the ability to do this:
`import LoginController from '@controllers/LoginController;`

File: src/view/dashboard/jobsearchcomponents/JobList.jsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+import {useSelector} from 'react-redux';
+
+import JobController from '../../../controllers/JobController.js';
+import {selectJobList} from '../../../model/state/selectors.js';
+
+import Job from './Job.jsx';
+
+const JobList = () => {
+    const jobList = useSelector(selectJobList);
+    // console.log('list', jobList)
```
Comment (alexparker) [inline src/view/dashboard/jobsearchcomponents/JobList.jsx:11]: Get in the practice of recognizing and not committing this kind of stuff.

File: src/index.css (hunk 2 above)
Comment (Cybersck) [reply src/index.css:13]: This should be removed completely, its replaced with App.css - I'm looking through all of your comments and will make the necessary changes.

File: src/view/dashboard/containers/JobListings.jsx (hunk 4 above)
Comment (Cybersck) [reply src/view/dashboard/containers/JobListings.jsx:4]: I didn't know about this, thanks for the tip! That's gonna look so much better!

File: src/view/dashboard/containers/JobTracker.jsx (hunk 5 above)
Comment (wildcard329) [reply src/view/dashboard/containers/JobTracker.jsx:66]: This was not an intended deletion. I have been working on features for next release canvas, but did not want to have to test them on this one.

Comment (wildcard329) [reply src/view/dashboard/containers/JobTracker.jsx:66]: If it is still considered bad practice I will be more careful of this in the future.

**Pull request closed without merging**